packages/thirdweb/src/wallets/in-app/core/wallet/in-app-core.ts

```diff
@@ -48,6 +57,13 @@ export function createInAppWallet(args: {
     getChain: () => chain,
     getConfig: () => createOptions,
     getAccount: () => account,
+    getProfiles: async () => {
+      if (!client) {
+        return [];
+      }
```

Comment:
should this be a legal case, maybe this should throw?

Reply:
If they haven't connected a wallet and they try to get profiles, I figured it should return a empty rather than throw. Saves them a try-catch if they're dealing with a wallet of unknown connection status